examples/uuid/src/main.rs

```diff
@@ -8,21 +8,21 @@ extern crate rocket;
 extern crate rocket_contrib;
 
 use std::collections::HashMap;
-use uuid::Uuid;
-use rocket_contrib::UUID;
+use uuid::Uuid as RealUuid;
```

Comment:
Don't import this as `RealUuid`. Instead use `uuid::Uuid` as needed in the example.